.circleci/config.yml

```diff
@@ -7028,6 +7031,28 @@ workflows:
           build_environment: "pytorch-vulkan-linux-bionic-py3.6-clang9-test"
           docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-bionic-py3.6-clang9"
           resource_class: large
+      - pytorch_linux_build:
+          name: pytorch_linux_bionic_cuda10_2_cudnn7_py3_9_gcc9_build
```

Comment:
seems like cuda10 is not compatible with gcc9. maybe up this to cuda11? and remove the test job?

Reply:
I've already downgraded gcc requirement to gcc7. And I don't want to move CI to cuda-11, since it's just slower to build and would not give us any extra signal in CI (and we need to test python-3.9, as I assume there are number of pending issues with the code)